src/MoneyAdmin.Domain.Core/Commands/CommandResult.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Threading.Tasks;
+
+namespace MoneyAdmin.Domain.Core.Commands
+{
+    public class CommandResult
+    {
+
+        public CommandResult(Exception exception) : this()
```

Comment:
Use expression body follow the [documention](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members)